# Franka Arm 목표 (ROS2 -> shm)
# Arm_Num = 2 (Left, Right)
# Arm_DOF = 7

float64[7] joint_targets    # Arm_j_tar
uint8 arm_id                # 0=Right, 1=Left

